.github/workflows/examples.yml

```
@@ -16,79 +16,79 @@
+          path: vendor/bundle
+          key: package-app-gem-cache-${{ hashFiles('react_on_rails.gemspec') }}-${{ hashFiles('Gemfile.development_dependencies') }}-${{ matrix.versions }}
+      - id: get-sha
+        run: echo "::set-output name=sha::$(git rev-parse HEAD)"
+      - name: Install Node modules with Yarn for renderer package
+        run: |
+          yarn install --no-progress --no-emoji
+          sudo yarn global add yalc
+      - name: yalc publish for react-on-rails
+        run: yalc publish
+      - name: Install Ruby Gems for package
+        run: bundle lock --add-platform 'x86_64-linux' && bundle check --path=vendor/bundle || bundle  _2.5.9_ install --path=vendor/bundle --jobs=4 --retry=3
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2015:info:1:43: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)